Makefile

```diff
@@ -0,0 +1,74 @@
+# Makefile for Teia Smart Contracts
+#
+# Usage example:
+#
+#   TARGET=daoToken make compile test 
+#
+
+
+# Location of SmartPy CLI (adjust to local installation, or set environment variable)
+SMARTPY:=~/ext/smartpy-cli/SmartPy.sh
```

Comment:
Could we have here the directory that smartpy uses by default?
~/smartpy-cli/SmartPy.sh
In this way it will agree with what we have in the README

Reply:
Of course! It now looks at the default install location unless SMARTPY_DIR is set in the shell environment variables.